cmd/script_exporter/script_exporter.go

```diff
@@ -170,11 +170,6 @@ func metricsHandler(w http.ResponseWriter, r *http.Request) {
 	// Get script from url parameter
 	params := r.URL.Query()
 	scriptName := params.Get("script")
```

Comment:
Is this still necessary?

Reply:
Not really now but I left them in as I wasn't too confident dropping the whole queryParam parsing, also it might a good idea to leave them in just in case someone wants to use that behaviour, but I guess if so they can use the original code.
So I've removed it now.

Reply:
Could also drop the whole queryParams parsing though that might break some things and is a more evasive change but it should be simple enough and I can test on live anyway so looking at that.